db/commitlog/commitlog.cc

```diff
@@ -796,10 +806,12 @@ class db::commitlog::segment : public enable_shared_from_this<segment>, public c
     // The commit log entry overhead in bytes (int: length + int: head checksum)
     static constexpr size_t entry_overhead_size = 2 * sizeof(uint32_t);
     static constexpr size_t multi_entry_overhead_size = entry_overhead_size + sizeof(uint32_t);
+    static constexpr size_t fractured_entry_overhead_size = 4 * sizeof(uint32_t);
```

Comment:
nit: s/fractured/fragmented/

also, I'm not sure if the entry header extends the multi_entry header, but if so, maybe this better be represented as `multi_entry_overhead_size + sizeof(uint32_t)`.

Reply:
No, the two are different. Multi-entry is an _additional_, wrapping entry, that is slightly bigger. The frag overhead is an _additional_ overhead, added to "normal" entry. While the calculating could be done with frag_overhead including the initial entry overhead, it would make the math/logic less readable imho, so I am opting for this...